Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ExternalCalls.aspx.cs

```diff
@@ -20,6 +20,11 @@ public partial class ExternalCalls : System.Web.UI.Page
         private const string ConnectionString = @"Data Source=.\SQLEXPRESS;Initial Catalog=RDDTestDatabase;Integrated Security=True";
 
         /// <summary>
+        /// Invalid connection string to database.
+        /// </summary> 
+        private const string InvalidConnectionString = @"Data Source=invalid\SQLEXPRESS;Initial Catalog=RDDTestDatabase;Integrated Security=True";
```

Comment:
I've added incorrect credentials case. Do you think that we really need to test many different scenarios in functional tests? We covered instrumentation and information retrieval for two diff cases (agree that one was not enough), but it's not the goal to cover all possible exceptions from Open...